src/main/flight/mixer.c

```diff
@@ -296,7 +296,11 @@ static FAST_RAM_ZERO_INIT float idleThrottleOffset;
 static FAST_RAM_ZERO_INIT float idleMinMotorRps;
 static FAST_RAM_ZERO_INIT float idleP;
 #endif
-
+#if defined(USE_BATTERY_VOLTAGE_SAG_COMPENSATION)
+static FAST_RAM_ZERO_INIT float vbatSagCompensationFactor;
+static FAST_RAM_ZERO_INIT float vbatFull;
```

Comment:
There's no reason for this to be a static. It's simply assigned the const `CELL_VOLTAGE_FULL_CV` in `mixerInitProfile()` and never is reassigned after that.